### PR TITLE
Create BLorient11764.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
New LIT IDs for ms_i3, ms_i4, ms_i5 and ms_i6 have been created after discussion in https://github.com/BetaMasaheft/Documentation/issues/2594, https://github.com/BetaMasaheft/Documentation/issues/2596, https://github.com/BetaMasaheft/Documentation/issues/2597 and https://github.com/BetaMasaheft/Documentation/issues/2598.

It would be nice to have an art theme for magic pictures, but I assume, that there is none. So I marked them up simply as "miniatures".

### CODE REVIEW

**Review by karljonaskarlsson — APPROVED** [review]

**Comment by eu-genia** [comment]
Most "magic pictures" I know are schematic drawings, not miniatures.

File: LondonBritishLibrary/orient/BLorient11764.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient11764.xml:144]: keywords

File: LondonBritishLibrary/orient/BLorient11764.xml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                            </supportDesc> 
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Square, rather careful script.</desc>
+                                <date notBefore="1600" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient11764.xml:87]: drawing

File: LondonBritishLibrary/orient/BLorient11764.xml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                                <desc>Coloured magic picture.</desc>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient11764.xml:88]: you can mark up magic as a term, then one could search for all drawings containing the keyword in our filtered search

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by nafisa-valieva — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**